### PR TITLE
Changing tracking uri to azure_ai_project

### PR DESCRIPTION
**Pull request by singankit**
# Description

Please add an informative description that covers that changes made by the pull request and link all relevant issues.

# All Promptflow Contribution checklist:
- [ ] **The pull request does not introduce [breaking changes].**
- [ ] **CHANGELOG is updated for new features, bug fixes or other significant changes.**
- [ ] **I have read the [contribution guidelines](../CONTRIBUTING.md).**
- [ ] **Create an issue and link to the pull request to get dedicated review from promptflow team. Learn more: [suggested workflow](../CONTRIBUTING.md#suggested-workflow).**

## General Guidelines and Best Practices
- [ ] Title of the pull request is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### Testing Guidelines
- [ ] Pull request includes test coverage for the included changes.

### CODE REVIEW

**Review by nagkumar91 — DISMISSED** [review]

File: src/promptflow-evals/promptflow/evals/evaluate/_evaluate.py [hunk 1]
```diff
@@ -56,9 +56,9 @@ def _validate_and_load_data(target, data, evaluators, output_path, tracking_uri,
         if not isinstance(output_path, str):
             raise ValueError("output_path must be a string.")
 
-    if tracking_uri is not None:
-        if not isinstance(tracking_uri, str):
-            raise ValueError("tracking_uri must be a string.")
+    if azure_ai_project is not None:
+        if not isinstance(azure_ai_project, Dict):
+            raise ValueError("azure_ai_project must be a string.")
```
Comment (ninghu) [inline src/promptflow-evals/promptflow/evals/evaluate/_evaluate.py:61]: must be a dict

File: src/promptflow-evals/promptflow/evals/evaluate/_evaluate.py [hunk 2]
```diff
@@ -209,7 +212,7 @@ def evaluate(
     data: Optional[str] = None,
     evaluators: Optional[Dict[str, Callable]] = None,
     evaluator_config: Optional[Dict[str, Dict[str, str]]] = {},
-    tracking_uri: Optional[str] = None,
+    azure_ai_project: Optional[Dict] = None,
```
Comment (ninghu) [inline src/promptflow-evals/promptflow/evals/evaluate/_evaluate.py:215]: can we modify the e2e test to cover this?

Comment (singankit) [reply src/promptflow-evals/promptflow/evals/evaluate/_evaluate.py:215]: Done

File: src/promptflow-evals/promptflow/evals/evaluate/_evaluate.py (hunk 1 above)
Comment (singankit) [reply src/promptflow-evals/promptflow/evals/evaluate/_evaluate.py]: Done

**Review by ninghu — APPROVED** [review]

**Review by nagkumar91 — APPROVED** [review]

**Review by ninghu — APPROVED** [review]

**Pull request merged**